src/resources/assets/useUserAsset.ts

```diff
@@ -26,7 +25,7 @@ export function useUserAsset(uniqueId: string) {
 export function useUserNativeNetworkAsset(chainId: ChainId) {
   const network = getNetworkFromChainId(chainId);
   const { nativeCurrency } = getNetworkObj(network);
-  const { mainnetAddress, address } = nativeCurrency;
-  const uniqueId = mainnetAddress ? `${mainnetAddress}_${Network.mainnet}` : `${address}_${network}`;
+  const { address } = nativeCurrency;
+  const uniqueId = `${address}_${network}`;
```

Comment:
seems like this is the correct way of building this uniqueId for the `useUserAsset` hook

Reply:
it was never finding the polygon native asset